integrations/jina/src/haystack_integrations/components/embedders/jina/document_embedder.py

```diff
@@ -19,13 +19,15 @@ class JinaDocumentEmbedder:
 
     Usage example:
     ```python
+    import os
     from haystack import Document
-    from jina_haystack import JinaDocumentEmbedder
-
-    doc = Document(content="I love pizza!")
+    from haystack_integrations.components.embedders.jina import JinaDocumentEmbedder
 
+    os.environ("JINA_API_KEY") = "YOUR_JINA_API_KEY"
```

Comment:
Let's remove this and also the os import. While I understand that we want to explain that there is a JINA_API_KEY, this code is not the way we want users to use Haystack. We have other ways to explain that there is a JINA_API_KEY needed.

Reply:
done